### PR TITLE
feat: Adding tilt lsp client

### PR DESCRIPTION
**Pull request by Konubinix**

### CODE REVIEW

File: clients/lsp-tilt.el [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+;;; Commentary:
+
+;; Using tilt mode from https://github.com/Konubinix/tilt-mode
+
+;;; Code:
+
+(require 'lsp-mode)
+
+(add-to-list 'lsp-language-id-configuration '(tiltfile-mode . "tiltfile"))
```
Comment (yyoncho) [inline clients/lsp-tilt.el:29]: we typically do that in lsp-mode.el . Also add it in  `lsp-client-packages` to be loaded when lsp-mode starts.

**Review by yyoncho — COMMENTED** [review]
Looks good. One minor nit.

File: clients/lsp-tilt.el (hunk 1 above)
Comment (Konubinix) [reply clients/lsp-tilt.el]: I did it. Is it ok that way?

**Comment by Konubinix** [comment]
Is there anything more to be done before this can be approved?

**Comment by Konubinix** [comment]
I rebased the code on top of master

**Review by yyoncho — APPROVED** [review]

**Pull request merged**